chart/README.md

```diff
@@ -349,6 +350,8 @@ helm install airflow . \
     --set workers.persistence.enabled=false
 ```
 
+You can also configure the number of tasks that each keda worker will take with the `config.celery.worker_concurrency` value.
```

Comment:
Word choice might be a little imprecise here.

They are _celery_ workers not _keda_ workers --- keda just autoscales the celery workers but does not have workers of its own.

Reply:
Agree.
Would that be better?

> KEDA scaling (ie. the number of Airflow celery workers that are spawned by KEDA) will also depend on the number of task each worker is allowed to handle. This number is the same as the `worker_concurrency` Airflow parameter and it must be tweaked with the `config.celery.worker_concurrency` value to properly propagate to the KEDA scaler configuration.

Reply:
OK I see, the important point you are making here is that you need to set celery concurrency through helm in order to have KEDA scaling be consistent with celery config.  E.g. if you use airflow.cfg, orr env vars, they won't necessarily agree.

I might just include the query template to make it explicit.

Here's another crack at it:

> KEDA will derive the desired number of celery workers by querying the metastore: <the query>.  You should set celery worker concurrency through helm value `config.celery.worker_concurrency` (i.e. instead of airflow.cfg or environment variables) so that the KEDA trigger will be consistent with worker concurrency.

Reply:
Awesome, thanks for the suggestion, let's go for this!